requirements_tests.txt

```diff
@@ -0,0 +1,5 @@
+pytest
```

Comment:
Please remove this file. For tests we can use the development requirements.

Reply:
Removed.